mono/metadata/class.c

```diff
@@ -15,41 +15,41 @@
 #include <alloca.h>
 #endif
 #include <glib.h>
-#include <stdio.h>
-#include <string.h>
-#include <stdlib.h>
-#include <mono/metadata/image.h>
-#include <mono/metadata/image-internals.h>
-#include <mono/metadata/assembly.h>
+#include <mono/metadata/appdomain.h>
 #include <mono/metadata/assembly-internals.h>
-#include <mono/metadata/metadata.h>
-#include <mono/metadata/metadata-internals.h>
-#include <mono/metadata/profiler-private.h>
-#include <mono/metadata/tabledefs.h>
-#include <mono/metadata/tokentype.h>
+#include <mono/metadata/assembly.h>
+#include <mono/metadata/attrdefs.h>
 #include <mono/metadata/class-internals.h>
-#include <mono/metadata/object.h>
-#include <mono/metadata/appdomain.h>
-#include <mono/metadata/mono-endian.h>
+#include <mono/metadata/custom-attrs-internals.h>
 #include <mono/metadata/debug-helpers.h>
-#include <mono/metadata/reflection.h>
 #include <mono/metadata/exception.h>
-#include <mono/metadata/security-manager.h>
-#include <mono/metadata/security-core-clr.h>
-#include <mono/metadata/attrdefs.h>
 #include <mono/metadata/gc-internals.h>
-#include <mono/metadata/verify-internals.h>
+#include <mono/metadata/image-internals.h>
+#include <mono/metadata/image.h>
+#include <mono/metadata/metadata-internals.h>
+#include <mono/metadata/metadata.h>
 #include <mono/metadata/mono-debug.h>
-#include <mono/metadata/custom-attrs-internals.h>
+#include <mono/metadata/mono-endian.h>
+#include <mono/metadata/object.h>
+#include <mono/metadata/profiler-private.h>
+#include <mono/metadata/reflection.h>
+#include <mono/metadata/security-core-clr.h>
+#include <mono/metadata/security-manager.h>
+#include <mono/metadata/tabledefs.h>
+#include <mono/metadata/tokentype.h>
+#include <mono/metadata/verify-internals.h>
+#include <mono/utils/atomic.h>
+#include <mono/utils/bsearch.h>
+#include <mono/utils/checked-build.h>
 #include <mono/utils/mono-counters.h>
-#include <mono/utils/mono-string.h>
 #include <mono/utils/mono-error-internals.h>
 #include <mono/utils/mono-logger-internals.h>
 #include <mono/utils/mono-memory-model.h>
-#include <mono/utils/atomic.h>
+#include <mono/utils/mono-string.h>
 #include <mono/utils/unlocked.h>
-#include <mono/utils/bsearch.h>
-#include <mono/utils/checked-build.h>
+#include <stdio.h>
```

Comment:
Can we avoid the head reorganization and/or Clang formatting? Stuff like this will make upstream merging or rebasing impossible.

Reply:
yeap, I'll get rid of it, that's one of the reasons I left it as preview from the get go, was really torn about the clang formater doing this stuff all over the place 👎